targets/targets.json

```diff
@@ -696,6 +696,29 @@
         "device_name": "MK64FN1M0xxx12",
         "bootloader_supported": true
     },
+    "RAPIDIOT": {
+        "inherits": ["Target"],
+        "core": "null",
+        "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
+        "macros": ["FSL_RTOS_MBED", "USE_EXTERNAL_RTC"],
+        "default_toolchain": "ARM",
+        "default_lib": "std",
+        "release_versions": ["2", "5"]
+    },
+    "RAPIDIOT_K64F": {
+        "inherits": ["RAPIDIOT"],
+        "core": "Cortex-M4F",
+        "extra_labels": ["Freescale", "MCUXpresso_MCUS", "KSDK2_MCUS", "MCU_K64F"],
+        "macros_add": ["CPU_MK64FN1M0VMD12", "TARGET_K64F"],
+        "is_disk_virtual": true,
+        "mbed_rom_start": "0x00014000",
+        "mbed_rom_size": "0xEC000",
```

Comment:
@0xc0170, @mmahadevan108 / @maclobdell  : Would it be better to move these 2 (ram,rom starts) to the respective linker descriptions (ARMCC / GCC / IAR) rather than here? wrapped in an #ifdef for the rapidiot targets only..

Reply:
https://github.com/ARMmbed/mbed-os/blob/master/tools/config/__init__.py#L628 - cant find the documentation so the code reference.